DQMOffline/Trigger/src/HLTMuonMatchAndPlot.cc

```diff
@@ -374,37 +376,35 @@ vector<size_t> HLTMuonMatchAndPlot::matchByDeltaR(const vector<T1>& collection1,
 
 MuonCollection HLTMuonMatchAndPlot::selectedMuons(const MuonCollection& allMuons,
                                                   const BeamSpot& beamSpot,
-                                                  bool hasRecoCuts,
-                                                  const StringCutObjectSelector<reco::Muon>& selector,
+                                                  double RecoMuonEtaMax,
+                                                  double RecoMuonEtaMin,
+                                                  bool IsMuonGlb,
                                                   double d0Cut,
                                                   double z0Cut) {
-  // If there is no selector (recoCuts does not exists), return an empty collection.
-  if (!hasRecoCuts)
-    return MuonCollection();
-
   MuonCollection reducedMuons;
   for (auto const& mu : allMuons) {
     const Track* track = nullptr;
     if (mu.isTrackerMuon())
       track = &*mu.innerTrack();
     else if (mu.isStandAloneMuon())
       track = &*mu.outerTrack();
-    if (track && selector(mu) && fabs(track->dxy(beamSpot.position())) < d0Cut &&
-        fabs(track->dz(beamSpot.position())) < z0Cut)
+    bool muID = true;
+    if (IsMuonGlb)
+      muID = mu.isGlobalMuon();
+    else
+      muID = mu.isStandAloneMuon();  // minimun ID (requested for cosmics)
```

Comment:
This could be made more compact:
```suggestion
    // minimun ID (requested for cosmics) is being a StandAlone muon
    bool muID = IsMuonGlb? mu.isGlobalMuon() : mu.isStandAloneMuon(); 
```